app/presenters/result_set_presenter.rb

```diff
@@ -2,11 +2,11 @@ class ResultSetPresenter
   include ERB::Util
   include ActionView::Helpers::NumberHelper
 
-  attr_reader :pluralised_document_noun, :debug_score, :start_offset, :include_ecommerce
+  attr_reader :pluralised_document_noun, :debug_score, :start_offset, :include_ecommerce, :include_ga4_ecommerce_tracking
```

Comment:
If there's a requirement that we track/don't track some of the finders in GA4 then ignore me, but if we're going to track all of them then I wonder if we should just hard code that? Might make the code a bit simpler - wouldn't have to initialise and pass around variables, and the tests could be simpler.

Reply:
Thanks Andy, I'll remove the GA4 variable